Solutions/BitSight/Parsers/BitSightBreaches.txt

```diff
@@ -0,0 +1,44 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias (e.g. BitSightBreaches).
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. BitSightBreaches | take 10).
+// References : 
+// Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// Tech Community Blog on KQL Functions : https://techcommunity.microsoft.com/t5/Azure-Sentinel/Using-KQL-functions-to-speed-up-analysis-in-Azure-Sentinel/ba-p/712381
+//
+let Breaches_view  = view () { 
+    Breaches_data_CL
+    | extend 
+                EventVendor ="BreachesBitSight",
+                EventProduct ="BreachesBitSight",
+                GUID = column_ifexists('guid_d', ''),
```

Comment:
It can be guid_g as well. Consider both in this case.

Reply:
@devikamehra Changes done as above.